src/device/ping-time-linux.cpp

```diff
@@ -0,0 +1,17 @@
+#include <ping-time.h>
+
+#include <chrono>
+#include <unistd.h>
+
+int PingTime::timeMs() {
+  // std::chrono::time_point
+  auto now = std::chrono::high_resolution_clock::now();
+
+  // std::chrono::duration
+  auto duration    = now.time_since_epoch();
```

Comment:
what is the style of this project ? AFAIK avoid extra tailing spaces.

Reply:
llvm clang format

Reply:
I took away the spaces.